_includes/components/sidebar.html

```diff
@@ -1,7 +1,7 @@
 <div class="side-bar">
-  <div class="site-header">
+  <div class="site-header" role="banner">
     <a href="{{ '/' | relative_url }}" class="site-title lh-tight">{% include title.html %}</a>
-    <a href="#" id="menu-button" class="site-button">
+    <a href="#" id="menu-button" class="site-button" aria-label="Open menu">
```

Comment:
This would be better as a semantic `<button>` element since it takes an action rather than navigating somewhere. Not sure if that is in scope for this PR but it would be a great upgrade 🙏

Reply:
@mattxwang what do you think of this suggestion by Lindsey from our accessibility team?

Reply:
Thanks for chiming in Lindsey; I agree that either a `<button>` (or a combination of `role="button"` and some sort of `aria-pressed` attribute) would be appropriate. If you have a change in mind, go for it (you may have to slightly change the `#menu-button` or `site-button` CSS to make this not a visual change); alternatively, we can table this for a different issue/PR. Let me know what you think!

Reply:
Whatever is easiest works for me, thank you!!